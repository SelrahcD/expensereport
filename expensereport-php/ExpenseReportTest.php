<?php

use PHPUnit\Framework\TestCase;

require __DIR__ . '/vendor/autoload.php';

class ExpenseReportTest extends TestCase
{
    public function testExpenseReport(): void {
        
    }

}
