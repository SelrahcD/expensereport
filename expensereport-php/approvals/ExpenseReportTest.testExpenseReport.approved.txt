array (
)
####
Expense Report YYYY-MM-DD HH:MM:SSxm
Meal Expenses: 0
Total Expenses: 0

array (
  0 => 
  Expense::__set_state(array(
     'type' => 1,
     'amount' => 12,
  )),
)
####
Expense Report YYYY-MM-DD HH:MM:SSxm
Dinner	12	 
Meal Expenses: 12
Total Expenses: 12

array (
  0 => 
  Expense::__set_state(array(
     'type' => 2,
     'amount' => 12,
  )),
)
####
Expense Report YYYY-MM-DD HH:MM:SSxm
Breakfast	12	 
Meal Expenses: 12
Total Expenses: 12

array (
  0 => 
  Expense::__set_state(array(
     'type' => 3,
     'amount' => 12,
  )),
)
####
Expense Report YYYY-MM-DD HH:MM:SSxm
Car Rental	12	 
Meal Expenses: 0
Total Expenses: 12

array (
  0 => 
  Expense::__set_state(array(
     'type' => 1,
     'amount' => 12,
  )),
  1 => 
  Expense::__set_state(array(
     'type' => 1,
     'amount' => 45,
  )),
)
####
Expense Report YYYY-MM-DD HH:MM:SSxm
Dinner	12	 
Dinner	45	 
Meal Expenses: 57
Total Expenses: 57

array (
  0 => 
  Expense::__set_state(array(
     'type' => 1,
     'amount' => 5000,
  )),
)
####
Expense Report YYYY-MM-DD HH:MM:SSxm
Dinner	5000	 
Meal Expenses: 5000
Total Expenses: 5000

